packages/block-library/src/image/edit.js

```diff
@@ -351,6 +351,11 @@ export function ImageEdit( {
 					!! metadata?.bindings?.url &&
 					( ! blockBindingsSource ||
 						blockBindingsSource?.lockAttributesEditing ),
+				lockUrlControlsMessage: sprintf(
+					/* translators: %s: Label of the bindings source if exists */
+					__( 'Connected to %s' ),
+					blockBindingsSource?.label || __( 'dynamic data' )
+				),
```

Comment:
@swissspidy Could you confirm if this way of translating the string is correct, please? 🙂

Reply:
Are we still in the window in the 6.5 release cycle when we can add new translations?

Reply:
@gziolo Yes, string freeze only takes effect after RC 1.

`Connected to %s` is OK if there is a label.

`dynamic data` as a fallback like this is not OK. I recommend something like this:

```
blockBindingsSource?.label ? sprintf(
		 /* translators: %s: Label of the bindings source if exists */
					__( 'Connected to %s' ),
					blockBindingsSource?.label )
:  __( 'Connected to dynamic data' )
```

Question: where is `blockBindingsSource?.label` being translated?

Reply:
Most likely here:

https://github.com/WordPress/gutenberg/blob/a65b156238a514a7fc03c3262f7cac4091d38fb5/packages/editor/src/bindings/post-meta.js#L14

Reply:
Eventually, we will pass it from the server from the registry for sources.

Reply:
Thanks for the feedback 🙂 I just applied the suggested changes in [this commit](https://github.com/WordPress/gutenberg/pull/59434/commits/8741ca90db2de0bf66662fd8070e529f4be4c507).

Reply:
Looks good!